FlowCrypt/Controllers/Settings/Settings List/SettingsViewController.swift

```diff
@@ -31,17 +31,35 @@ final class SettingsViewController: TableNodeViewController {
             case .experimental: return "settings_screen_experimental".localized
```

Comment:
On line 28 there is a typo: `atteseter` should be `attester` - please help me fix. Thanks!

Reply:
done